actions/wait-for-workflow-job/README.md

```diff
@@ -0,0 +1,33 @@
+# wait-for-workflow-job
+
+Waits for a job in a workflow to complete. This is useful when you need to wait
+for a specific (required) job to complete but do not need the result of the
+whole workflow itself.
+
+```yaml
+name: example
+
+on:
+  merge_group:
+  pull_request:
+
+jobs:
+  run-job:
+    name: Wait For Job
+    runs-on:
+      labels: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
```

Comment:
Should the `actions: read` necessity be here, and called out here?

Reply:
I think he was thinking of using GATI for reading other repos?

in which case maybe we can add the `setup-gh-token` action into this readme too?

Reply:
I think this is just copy paste error. We can guess what permissions their job will need entirely so it may be more confusing to have any in the readme example. Will clean it out.

Reply:
Cleaned up in: https://github.com/smartcontractkit/.github/pull/454/commits/e02e6b59110bbf3ca6c14a23d1fa1641d2b0bde0